helm/soperator-crds/templates/slurmcluster-crd.yaml

```diff
@@ -1468,6 +1468,76 @@ spec:
                       with ssh keys for SSHD server
                     type: string
                 type: object
+              slurmConfig:
+                description: SlurmConfig represents the Slurm configuration in slurm.conf.
+                  Not all options are supported.
+                properties:
+                  completeWait:
+                    default: 0
+                    description: The time to wait, in seconds, when any job is in
+                      the COMPLETING state before any additional jobs are scheduled.
+                    format: int32
+                    type: integer
+                  debugFlags:
+                    default: ""
```

Comment:
Let's set it to `Cgroup,CPU_Bind,Gres,JobComp,Priority,Script,SelectType,Steps,TraceJobs` by default